modules/microshift-config-yaml.adoc

```diff
@@ -6,17 +6,22 @@
 [id="microshift-config-yaml_{context}"]
 = Using a YAML configuration file
 
-At start up, {microshift-short} checks the system-wide `/etc/microshift/` directory for a configuration file named `config.yaml`. If the configuration file does not exist in the directory, the built-in default values are used to start the service.
+At start up, {microshift-short} checks the system-wide `/etc/microshift/` directory for a configuration file named `config.yaml`. If the configuration file does not exist in the directory, built-in default values are used to start the service.
 
 [id="microshift-yaml-custom_{context}"]
 == Custom settings
-To create custom configurations, you must create a `config.yaml` file in the `/etc/microshift/` directory, and then change any settings that are expected to override the defaults before starting or restarting {microshift-short}.
+To create custom configurations, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`. Keep this file in the `/etc/microshift/` directory, and then you can change supported settings that are expected to override the defaults before starting or restarting {microshift-short}.
 
 [IMPORTANT]
 ====
 Restart {microshift-short} after changing any configuration settings to have them take effect. The `config.yaml` file is read only when {microshift-short} starts.
 ====
 
+[IMPORTANT]
+====
+Applications and other optional services used with your {microshift-short} cluster might also need to be restarted separately to apply configuration changes throughout the cluster. For example, when making changes to certain networking settings, you must stop and restart service and application pods to apply those changes.
+====
```

Comment:
As a pure tech writing rule, I tend to avoid stacked admonitions. I wonder if these two can be combined?

```suggestion
[IMPORTANT]
====
* Restart {microshift-short} after changing any configuration settings to have them take effect. The `config.yaml` file is read only when {microshift-short} starts.

* Applications and other optional services used with your {microshift-short} cluster might also need to be restarted separately to apply configuration changes throughout the cluster. For example, when making changes to certain networking settings, you must stop and restart service and application pods to apply those changes.
====
```

Reply:
> Applications and other optional services used with your {microshift-short} cluster might also need to be restarted separately

Do the docs tell users when applications and components need to be restarted in various procedures?

 ```suggestion
[IMPORTANT]
====
Restart {microshift-short} after changing any configuration settings to have them take effect. The `config.yaml` file is read only when {microshift-short} starts.
====

[IMPORTANT]
====
Applications and other optional services used with your {microshift-short} cluster might also need to be restarted separately to apply configuration changes throughout the cluster. For example, when making changes to certain networking settings, you must stop and restart service and application pods to apply those changes. A procedure will include a step to restart if required.
====
```

Reply:
No, that's for the sys admin to understand and decide. We can't know what they have running on MicroShift or how they configured it.

Reply:
Added "See each procedure for the task you are completing for more information." for those places where I know we say it (like multus and such where you have to delete pods to assign the new config).